docs/reference/operator-config.md

```diff
@@ -0,0 +1,7 @@
+---
+sidebar_position: 2
+---
+
+# Configuring Operator 
```

Comment:
there are already corresponding sections for this in `Getting Started`. We may want to name this just `Operator` and `Scaling Policies` respectively to make it clear that this is just a reference section not a "guide". At least that was my intention.